config.m4

```diff
@@ -24,19 +24,28 @@ if test "$PHP_MONGODB" != "no"; then
   dnl Check PHP version is compatible with this extension
   AC_MSG_CHECKING([PHP version])
 
-  if test -z "$PHP_CONFIG"; then
-    AC_MSG_ERROR([php-config not found])
-  fi
+  if test -z "${PHP_VERSION_ID}"; then
+    if test -z "$PHP_CONFIG"; then
+      AC_MSG_ERROR([php-config not found])
+    fi
 
-  PHP_MONGODB_PHP_VERSION=`${PHP_CONFIG} --version`
-  PHP_MONGODB_PHP_VERSION_ID=`${PHP_CONFIG} --vernum`
+    PHP_MONGODB_PHP_VERSION=`${PHP_CONFIG} --version`
+    PHP_MONGODB_PHP_VERSION_ID=`${PHP_CONFIG} --vernum`
+  else
+    PHP_MONGODB_PHP_VERSION="${PHP_VERSION}"
+    PHP_MONGODB_PHP_VERSION_ID="${PHP_VERSION_ID}"
+  fi
```

Comment:
I successfully ran `./configure` on Alpine using the `php:8.2-alpine` [Docker image](https://hub.docker.com/_/php). This still works, although it did reveal a bug in the build config output where we print "Bundled ()" when libmongocrypt isn't configured. Opened [PHPC-2386](https://jira.mongodb.org/browse/PHPC-2386) to track that.